framework/source/class/qx/ui/core/MExecutable.js

```diff
@@ -82,7 +82,6 @@ qx.Mixin.define("qx.ui.core.MExecutable",
      */
     _bindableProperties :
     [
-      "enabled",
```

Comment:
Btw - what's the usecase of assigning one Command to different Buttons at a time? Doesn't make sense IMHO.